handlers.go

```diff
@@ -57,6 +58,16 @@ var ConfigValidationHandler = Handler{
 	},
 }
 
+// ConfigValidationHandler is used to validate that required fields are
+// present. This validator is to be used when the jailer is turned off.
```

Comment:
nit on the comment: 

1/ LoadSnapshotConfigValidationHandler
2/ This comment mentions the validator is to be used when the jailer is turned off, is that accurate in the case of loading snapshots?

Reply:
Oh, I forgot to change this. No, this will just be called whenever LoadSnapshot is called. I'll investigate more on the jailer later, as I'm unsure how the jailer setup interacts with snapshotting.